transformations/dyslexia_words_swap/transformation.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import spacy
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class DyslexiaWordsSwap(SentenceOperation):
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+    keywords = ["lexical", "rule-based", "external-knowledge-based", "aural", "possible-meaning-alteration", "low-precision", "low-coverage", "low-generations"]
```

Comment:
Instead of a single line, write it in multiple lines.

Reply:
Fixed ! Thanks !